zip-XXXX.rst

```diff
@@ -0,0 +1,451 @@
+working to maintain clients.
+
+* One third of the remaining fee (25% of the total) shall be reserved for the
+  role of the "principal developer", a developer with additional voice in Zcash
+  governance. The principal developer allocation shall be capped similarly to
+  the Foundation's, based on the monthly volume-weighted average price.
+* The remaining two thirds of the fee (50% of the total), called the "outside
+  development fee", shall be distributed between at least two development teams,
+  chosen semi-annually by the Foundation, coinciding with network upgrades.
+  Unlike those of the Foundation and principal developer, these allocations
+  aren't affected by market conditions, and don't carry a burn requirement.
+
+Prior to each network upgrade, the Foundation shall recommend a list of
+addresses and a total number of ZEC per block each address is meant to receive
+from the dev fee. The recommendation will be "ratified" by the miners as the
+network upgrade activates.
+
+The role of dev fee recipients
+------------------------------
+
+Dev fee recipients are distinguished from grant recipients in the scope and
+timelines of their work, as well as the specificity of direction. The intent
+is to allow teams to focus on a core competency, while encouraging research and
+adjacent work.
+
+Dev fee recipients are chosen semi-annually by the Foundation based on their
+ability to move Zcash forward. Recipients will typically be development teams,
+though "full stack" teams that can communicate well with the community, expand
+Zcash usage, and widely share their work should be advantaged.
+
+Recipients shall submit quarterly plans to the community for their efforts, as
+well as monthly progress updates.
+
+All work funded by the dev fee will be open source, under licenses compatible
+with the existing Zcash clients.
+
+Though the Foundation shall periodically judge the efficacy of dev fee
+recipients, deciding on and driving roadmaps for Zcash is the role of dev fee
+recipients, in partnership with the community. This role is neatly balanced by
+users running full nodes and miners, either of which can veto a network upgrade.
+
+While dev fee recipients are not required to work exclusively on Zcash,
+considering the nature of their work, recipients must guarantee they aren't
+obliged to the interests of competing projects.
+
+The role of the principal developer
+-----------------------------------
+
+The role of the principal developer is as a "first among equals" amongst the dev
+fee recipients.
+
+The principal developer shall make a number of guarantees.
+
+1. Zcash shall be their exclusive focus, submitting financials periodically to
+   the Foundation as assurance.
+2. They shall maintain a well-run board and employ a qualified CFO.
+3. In addition to the existing open-source requirements, they shall agree to
+   assign any trademarks or patents relevant to Zcash to the Foundation.
```

Comment:
Yep. I think that's a poor idea because trademarks are use-it-or-lose-it. We need a single entity deciding to enforce, and deciding when to give up the :tm: